django/db/models/fields/__init__.py

```diff
@@ -224,6 +224,15 @@ def _check_field_name(self):
                     id='fields.E003',
                 )
             ]
+        elif self.name == 'exact':
+            return [
+                checks.Error(
+                    "'exact' is a reserved word that cannot be used as a field name.",
+                    hint=None,
+                    obj=self,
+                    id='fields.E008',
```

Comment:
Is new code needed? The sense of error is the same, I think you should use `fields.E003` here too.